com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -2294,6 +2328,15 @@ def print(self, tag=None, name=None, write_punctuations=None):
         If `tag` is not `None` then each tuple has "tag: " prepended
         to it before printing.
 
+        .. note::
+             Punctuation marks are in-band signals that are inserted between tuples in a stream. Window punctuations are inserted into a stream that are related to the semantics of the operator. One example is the :py:meth:`~Window.aggregate`, which inserts a window marker into the output stream after each aggregation. 
+             
+             There are two kinds of punctuation markers, that are written to stdout when `write_punctuations` is set to `True`:
```

Comment:
Suggestion: There are two kinds of punctuation markers, **which** are written to stdout ...